examples/tutorials/tutor.py

```diff
@@ -43,9 +43,10 @@ def main(root_dir):
         print(usage)
         sys.exit(1)
 
-    root_dir = sys.argv[1]
-    try:
-        main(root_dir)
-    except NameError as e:
-        print(e)
-        print(usage)
+    # Determine the root path to use for the tutorial files:
+    if len(sys.argv) == 2:
+        root_dir = sys.argv[1]
+    else:
+        root_dir = os.getcwd()
```

Comment:
I think you want relax the `if len(sys.argv) != 2` check on line 42 in order to get to this branch.

Reply:
Oh yes, you're right, I completely missed that.